### PR TITLE
Signoff: Content feedback fixes

### PR DESCRIPTION
**Pull request by arnoldsandoval-okta**

### CODE REVIEW

File: packages/docs/components/link.md [hunk 1]
```diff
@@ -65,25 +65,27 @@ Avoid using Links for actions, preferring Buttons instead.
...
   <template>
     <a href="#">View solar system</a>
   </template>
 </Visual>
 
-<Visual variant="negative">
+<Visual variant="smiley-neutral">
   <template>
-    <a href="#">View</a>
+    <a href="#">View solar</a>
```
Comment (edburyenegren-okta) [inline packages/docs/components/link.md:83]: solar system*

File: packages/docs/components/tab.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ see what could be grouped together. Those themes should become your Tabs.
 
 <Description>
 
-**Tabs are not navigation**. Meaning they don’t take you from place to place. Rather, they are meant for context switching related to the page.
+_Tabs are not navigation_. Meaning they don’t take you from place to place. Rather, they are meant for context switching related to the page.
```
Comment (edburyenegren-okta) [inline packages/docs/components/tab.md:83]: I'd prefer we left out most whole-sentence formatting. It's not present in most of the docs, and I'm not sure this is much stronger than our other guidelines.

File: packages/vuepress-theme-odyssey/styles/components/_DocsPagination.scss [hunk 3]
```diff
@@ -1,7 +1,14 @@
-.page-nav {
+.page-footer {
+  margin-top: $spacing-xl;
   padding: $spacing-m 0;
   border-top: 2px solid var(--border-color-display, #{$border-color-display});
+  text-align: center;
 
+  &,
```
Comment (edburyenegren-okta) [inline packages/vuepress-theme-odyssey/styles/components/_DocsPagination.scss:7]: are these double nested?

File: packages/vuepress-theme-odyssey/styles/docskit/_Visual.scss [hunk 4]
```diff
@@ -79,4 +79,16 @@
   &.is-docskit-visual-negative .docskit-visual--title::before {
     content: '🚫';
   }
+
+  &.is-docskit-visual-smiley-positive .docskit-visual--title::before {
+    content: '😄';
+  }
+
+  &.is-docskit-visual-smiley-neutral .docskit-visual--title::before {
+    content: '🙄';
```
Comment (edburyenegren-okta) [inline packages/vuepress-theme-odyssey/styles/docskit/_Visual.scss:88]: The :eyeroll: doesn't read at neutral to me; seems like the user is doing something wrong.

File: packages/docs/components/link.md (hunk 1 above)
Comment (arnoldsandoval-okta) [reply packages/docs/components/link.md:83]:
```suggestion
    <a href="#">View system</a>
```

File: packages/docs/components/tab.md (hunk 2 above)
Comment (arnoldsandoval-okta) [reply packages/docs/components/tab.md:83]:
```suggestion
Tabs are not navigation. Meaning they don’t take you from place to place. Rather, they are meant for context switching related to the page.
```

File: packages/vuepress-theme-odyssey/styles/docskit/_Visual.scss (hunk 4 above)
Comment (arnoldsandoval-okta) [reply packages/vuepress-theme-odyssey/styles/docskit/_Visual.scss:88]: @joejezowski-okta any recommendations here?

**Pull request closed without merging**